manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SERPRO.AssinadorSERPRO
+PackageVersion: 4.0.2
+PackageLocale: pt-BR
+Publisher: SERPRO- Serviço Federal de Processamento de dados
+PackageName: Assinador Serpro
+License: SERPRO
```

Comment:
https://github.com/microsoft/winget-pkgs/pull/99508/files#r1140284091
The publisher is not a license. If it is not an open source license like GPL-2.0 or MIT, it should be proprietary